highlight.io/components/QuickstartContent/traces/go/gorm.tsx

```diff
@@ -0,0 +1,48 @@
+import { siteUrl } from '../../../../utils/urls'
+import { goGetSnippet, initializeGoSdk } from '../../backend/go/shared-snippets'
+import { QuickStartContent } from '../../QuickstartContent'
+import { verifyTraces } from '../shared-snippets'
+
+export const GormTracesContent: QuickStartContent = {
+	title: 'Tracing with GORM',
+	subtitle:
+		'Learn how to set up auto-instrumented tracing for your database calls using the GORM library.',
+	logoUrl: siteUrl('/images/quickstart/go.svg'),
+	entries: [
+		goGetSnippet,
+		initializeGoSdk,
+		{
+			title: 'Initialize the GORM library with the Highlight hooks',
+			content:
+				'Import the Highlight GORM library and call the `SetupGORMTracing` hook with any attributes wanted for context.',
+			code: [
+				{
+					text: `import (
+	"github.com/highlight/highlight/sdk/highlight-go"
+  htrace "github.com/highlight/highlight/sdk/highlight-go/trace"
+	"go.opentelemetry.io/otel/attribute"
+)
+
+if err := htrace.SetupGORMTracing(db, attribute.String(highlight.ProjectIDAttribute, <YOUR_PROJECT_ID>)); err != nil {
```

Comment:
i think the order of this is confusing.
`htrace.SetupGORMTracing(db, attribute.String(highlight.ProjectIDAttribute, <YOUR_PROJECT_ID>))`
takes a `db` object, but it's not obvious where that comes from. isn't that the `db` you get from `db, err = gorm.Open([DB_SETTINGS])`?

if so, could we refactor this to be

step 1
```go
DB, err = gorm.Open([DB_SETTINGS])
if err := htrace.SetupGORMTracing(db, attribute.String(highlight.ProjectIDAttribute, <YOUR_PROJECT_ID>)); err != nil {
  // ...
}
```
step 2... Call GORM with the trace context
```go
DB.WithContext(ctx).Find(&user)
```

Reply:
Ohh great catch! I'll update that